tests/test_benchmark_feedin.py

```diff
@@ -231,7 +231,7 @@ def test_benchmark_feedin_tariff_optimize_positive_value(self, margs):
         optimized_added_cap = scalar_matrix[OPTIMIZED_ADD_CAP]["pv_plant_01"]
         assert (
             optimized_added_cap == 5000
```

Comment:
Maybe add a linke where there is a test that `installedCap == 0`? Otherwise the `optimized_add_cap` would not have to be 5000.

Reply:
@smartie2076 I didn't write this test, someone else did and it was already set like this. I changed the usage of maximum capacity in a string to maximum additional capacity. It says that there is a high positive feed-in tarrif and moderate specific PV costs, so it should choose the highest possible additional capacity right? Then the optimized_add_cap should equal 5000 because this is the maximum additional capacity available

Reply:
Yeah, I know that this was by some-one else, and the assertion is also correct. I just want to make sure that no `installedCap` is present in this benchmark test, so that the assertion is performed on the right grounds...

I similarily check that no-one edited the input files for the `AE-grid-battery` benchmark test in my new PR #859.

Reply:
It is more cherry on top then needed.

Reply:
and what effect does the installed capacity have on whether or not the assertion is on the right grounds? so should I change the whole test and use a different set of input files? Because another test uses 'Feedin_optimize'

Reply:
If `installedCap`>0, `optimized_add_cap!=maximumCap`, but `optimized_add_cap<=maximumCap-installedCap`.
You just need to add an
`assert scalar_matrix[INSTALLED_CAP]["pv_plant_01"] == 0`

Reply:
I will try to make clearer my understanding of the test (but I could have misunderstood!):

1. installed capacity = 3000, max capacity = 8000, max additional capacity = 5000
2. because the feed in tarrif for the energy system is so high but specific costs are so low, the system should install the maximum possible amount of solar
3. this means that the optimized additional capacity should be the maximum additional capacity (5000), so the assertion of 5000 for the optimzied additional capacity applies

And then because of this, the installed capacity does not have to be zero

Reply:
Ah, thats weird. In my local pulled branch the input files for the benchmark test are different and `MaximumCap=5000`, with an `InstalledCap`. Strange... o.O

You are completely right, all three arguments apply, the test of the coupled parameters is in there, and no additional assertion is needed.

Reply:
Hah? How can that be different? Your version is https://github.com/rl-institut/multi-vector-simulator/blob/feature/coupled_maximum_installed_capacities/tests/benchmark_test_inputs/Feedin_optimize/csv_elements/energyProduction.csv with the values you mentioned, but on dev my values are true: https://github.com/rl-institut/multi-vector-simulator/blob/dev/tests/benchmark_test_inputs/Feedin_optimize/csv_elements/energyProduction.csv. There is no commit that changes that file?!

Reply:
Hmm weird, I will make sure I've pushed the way that I mentioned. Thanks!

Reply:
I think 27bddec65f008607fdb59da00e5433824ea29584 changed it. No idea why it is not shown in the review.